doc/Type/IO/Spec/QNX.pod6

```diff
@@ -4,18 +4,19 @@
 
 =SUBTITLE Platform specific operations on file and directory paths QNX
 
-    class IO::Spec::QNX is IO::Spec { }
+    class IO::Spec::QNX is IO::Spec::Unix { }
 
-This sub-class of L<IO::Spec|/type/IO::Spec> specific to QNX will be
-available via C<$*SPEC> if the perl is running on QNX.
+An object of this type is available via the variable C<$*SPEC> if the
+Raku interpreter is running on a C<QNX> platform.
 
-B<NOTE:> the C<IO::Spec::*> classes provide low-level path operations. Unless
-you're creating your own high-level path manipulation routines, you don't
-need to use C<IO::Spec::*>. Use L«C<IO::Path>|/type/IO::Path» instead.
+The C<IO::Spec::*> classes provide low-level path operations. Unless
```

Comment:
Don't like repetition of text in the documentation. As long as you're improving this, maybe you should put all this text in a single place and link it.

Reply:
Sure, what about the update I pushed ? Would that be OK ?